helpers/integration_test.go

```diff
@@ -340,6 +341,38 @@ var _ = Describe("Pipeline Adapter", Ordered, func() {
 		Expect(result).To(BeFalse())
 	})
 
+	It("can handle malicious HACBS_TEST_OUTPUT result", func() {
+		var overflow bytes.Buffer
+		json_max_chars := 18446744073709551616
+		for i := 0; i < json_max_chars+100; i++ {
+			overflow.WriteString("A")
+		}
+		testpipelineRun.Status = tektonv1beta1.PipelineRunStatus{
+			PipelineRunStatusFields: tektonv1beta1.PipelineRunStatusFields{
+				TaskRuns: map[string]*tektonv1beta1.PipelineRunTaskRunStatus{
+					"task1": {
+						PipelineTaskName: "task-malformed-result",
+						Status: &tektonv1beta1.TaskRunStatus{
+							TaskRunStatusFields: tektonv1beta1.TaskRunStatusFields{
+								TaskRunResults: []tektonv1beta1.TaskRunResult{
+									{
+										Name:  "HACBS_TEST_OUTPUT",
+										Value: overflow.String(),
```

Comment:
Something like `Value: *tektonv1beta1.NewArrayOrString(overflow.String())` should work here.